components/Footer/index.jsx

```diff
@@ -1,9 +1,67 @@
-// TODO: Create a footer component that displays the footer of the website.
+"use client";
+import MSLogo from "../../public/images/logos/MSLogo.png";
+import FCSCLogo from "../../public/images/logos/FCSCLogo.png";
+import MLSALogo from "../../public/images/logos/MLSALogo.png";
+import Image from "next/image";
+import Link from "next/link";
+
+const socialMedia = [
+  { src: "/images/logos/Facebook.png", alt: "Facebook", link: "/" },
+  { src: "/images/logos/Instagram.png", alt: "Instagram", link: "/" },
+  { src: "/images/logos/Twitter.png", alt: "Twitter", link: "/" },
+  { src: "/images/logos/GitHub.png", alt: "GitHub", link: "/" },
+];
+
+const MSLogs = [
+  { src: MSLogo, alt: "MS Club Of SLIIT", link: "https://msclubsliit.org/" },
+  { src: FCSCLogo, alt: "FCSC LOGO", link: "https://msclubsliit.org/" },
+  {
+    src: MLSALogo,
+    alt: "MLSA Logo",
+    link: "https://studentambassadors.microsoft.com/",
+  },
+];
+
 export default function Footer() {
   return (
-    // ! Change as needed.
-    <main>
-      <h1 className="text-6xl font-bold">Footer</h1>
-    </main>
+    <footer className="bg-white py-8 shadow-md">
+      <hr className="my-6 border-gray-300 dark:border-gray-700" />
```

Comment:
Remove the dark prefixed class if there is no dark mode support or you can comment on the reason for using it.